src/WaveformAnalysis.cpp

```diff
@@ -340,241 +340,241 @@ double WaveformAnalysis::bessel(double *x, double *p){
   //  fitresult->prob      = TMath::Prob( ffitfunc->GetChisquare(), 30-5 );
   //  fitresult->fitstat   = fitstat;
   //}
-  else if( pmt.type == PTF::mPMT_REV0_PMT ){ /// Add a new PMT type for the mPMT analysis.
-
-    // Find the mininum bin between 2040.0ns (bin 255) and  2320.0ns (bin 290)
-    double min_bin = 2400;
-    double min_bini = 0;
-    double min_value = 1999.0;
-    //    for(int i = 280; i < 320; i++){
-    for(int i = 250; i < 320; i++){
-      double value = hwaveform->GetBinContent(i);
-      if(value < min_value){
-        min_value = value;
-        min_bin = hwaveform->GetBinCenter(i);
-	min_bini = i;
-      }
-    }
-
-    double fit_minx = min_bin - 40.0;
-    double fit_maxx = min_bin + 8.0*2.5;
-
-    int fitstat;
-    double amplitude;
-
-    // ellipitcall modified gaussian
-    if(pmt.channel >= 16){
-      if( ffitfunc == nullptr ) ffitfunc = new TF1("mygauss",funcEMG,fit_minx-30,fit_maxx+30,5);
-      ffitfunc->SetParameters( fitresult->amp, fitresult->mean, 8.0, 1.0, fitresult->ped );
-      ffitfunc->SetParNames( "Amplitude", "Mean", "Sigma", "exp decay", "Offset" );
+//   else if( pmt.type == PTF::mPMT_REV0_PMT ){ /// Add a new PMT type for the mPMT analysis.
+
```

Comment:
not really and important comments, but you can use */ and *\ instead of doing \\ at each line !